velox/functions/prestosql/tests/StringFunctionsTest.cpp

```diff
@@ -1748,6 +1748,31 @@ TEST_F(StringFunctionsTest, fromBase64) {
   EXPECT_THROW(fromBase64("YQ==="), VeloxUserError);
 }
 
+TEST_F(StringFunctionsTest, fromBase64Url) {
+  const auto fromHex = [&](std::optional<std::string> value) {
```

Comment:
Please move this function definition closer to its usage.

Reply:
It is close to the function definition, not sure what you meant. The function and test is within one place. Please clarify if it is not acceptable.